lupa/_lupa.pyx

```diff
@@ -460,6 +486,13 @@ cdef class LuaRuntime:
             lua.lua_settop(L, old_top)
             unlock_runtime(self)
 
+    def set_max_memory(self, max_memory):
```

Comment:
Not sure if we need this. Do you see a use case for changing the memory limit later on?

Reply:
I'm using this in the tests atm

Reply:
Using this to temporarily remove the limit to load trusted library and code and then restricting it once loading a user's code